packages/gestalt/src/Masonry.js

```diff
@@ -54,6 +54,7 @@ type Props<T> = {|
           from: number,
         }
       ) => void | boolean | {}),
+  rtlLayout?: boolean,
```

Comment:
nit: I'm always suspicious of whether booleans communicate their impact properly. How do you feel about naming this something along the lines of `layoutDirection: 'rtl' | 'ltr'` and then setting a `defaultProps` to `ltr`?

Reply:
+1 on this

Reply:
Sounds good, will rename as such